cursorless-talon/src/snippets.py

```diff
@@ -127,12 +132,13 @@ def cursorless_insert_snippet_by_name(name: str):
             ImplicitDestination(),
         )
 
-    def cursorless_insert_snippet(body: str):
+    def cursorless_insert_snippet(
+        body: str,
+        destination: Optional[CursorlessDestination] = ImplicitDestination(),
+        scope_type: Optional[str] = None,
```

Comment:
Hesitant to expose `scope_type` as a string here, as that precludes more interesting scope types, but I guess it's just mirroring what we already do in our snippet definition schema 🤷‍♂️

Reply:
Exactly. We could talk about this on a meetup?

Reply:
sure https://github.com/cursorless-dev/cursorless/labels/to%20discuss